src/somef/process_repository.py

```diff
@@ -574,7 +593,7 @@ class GithubUrlError(Exception):
     # print("The URL provided seems to be incorrect")
     pass
 
-
+# tpronk said on 2023-11-09: the function below doesn't seem to be invoked anywhere in SOMEF
```

Comment:
It may be legacy used for ML I will have a look and delete if appropriate after your PR

Reply:
Removed the note